elife/mysql-client.sls

```diff
@@ -6,3 +6,16 @@ mysql-package-list:
             - mysql-client
             - python-mysqldb # py2, Salt 2017.7.x
             - python3-mysqldb # py3, Salt 2018.3+
+
+    # on Ubuntu 16.04, in Salt 2018.3, using Python3, there is a problem passing a 
+    # bytestring as an argument to a function that expects a regular string.
+    # this patch decodes any bytestring arguments into a regular string
+    file.patch:
+        - name: /usr/lib/python3/dist-packages/MySQLdb/cursors.py
+        - source: salt://elife/config/usr-lib-python3-dist-packages-MySQLdb-cursors.py.patch
```

Comment:
where does this come from? If it has been downloaded needs a URL, if handwritten state it

Reply:
the patch? I made the patch. I'll make a note of it's origin